eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -244,6 +244,12 @@ jobs:
             - script: cmake --build . --target ${{ artifact.Path }}-docs
               workingDirectory: build
               displayName: Generate docs (${{ artifact.Name }}-docs)
+              condition: and(succeeded(), ne('${{parameters.ServiceDirectory}}', 'appconfiguration'))
```

Comment:
I'm assuming we have a bug on this one.

Reply:
Yes
When we get doc comments generated from Typespec, we can turn doxygen validation on: https://github.com/Azure/autorest.cpp/issues/191